src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -133,6 +133,7 @@ int enroll_tpm2(struct crypt_device *cd,
                 const void *volume_key,
                 size_t volume_key_size,
                 const char *device,
+                uint32_t handle,
```

Comment:
i'd also call this `seal_key` or `storage_key` or so (depending on how the cmdline switch is going to be called)

Reply:
ack, updated to `seal_key_handle` (since we'll add `seal_key_file` later also)